docs/how-to/explore/reveal-blast-radius.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Reveal blast radius
+sidebar_custom_props:
+  tags: [AWS, GCP, DigitalOcean, Kubernetes]
+---
+
+# How to reveal the blast radius of a resource
+
+## Introduction
+
+Resoto maintains an asset inventory of your infrastructure metadata. It not only collects the metadata of your resources, but also assembles the relationship between them.
+We use this information to traverse not only direct dependencies, but also dependencies of dependant resources - transitive dependencies.
```

Comment:
there's a lot of dependencies in this sentence....

I think you want to break this sentence up, and first define what transitive dependencies are.